decidim-proposals/app/cells/decidim/proposals/proposal_vote/show.erb

```diff
@@ -2,8 +2,8 @@
   <% if component_settings.participatory_texts_enabled? && from_proposals_list %>
     <%= render partial: "decidim/proposals/proposals/participatory_texts/proposal_votes_count", locals: { proposal: resource, from_proposals_list: true } %>
   <% else %>
-    <% progress ||= resource.proposal_votes_count || 0 %>
     <% total ||= resource.maximum_votes || 0 %>
+    <% progress ||= resource.proposal_votes_count || 0 %>
```

Comment:
I'll try to not introduce cosmetic changes in fixes, to make us easy to do the backports and diffs.

```suggestion
    <% progress ||= resource.proposal_votes_count || 0 %>
    <% total ||= resource.maximum_votes || 0 %>
```